src/backend/impl/src/controllers/proposal_controller.rs

```diff
@@ -76,15 +76,41 @@ impl<A: AccessControlService, L: LogService, P: ProposalService> ProposalControl
                 );
             }
             Err(e) => {
-                let _ = self
-                    .log_service
-                    .log_error(format!("{:?}", e), Some("sync_proposals".to_string()));
+                let _ = self.log_service.log_error(
+                    format!("Error syncing proposals ({})", e),
+                    Some("sync_proposals".to_string()),
+                );
             }
         }
     }
 
-    fn list_proposals(&self) -> Result<ListProposalsResponse, ApiError> {
-        self.proposal_service.list_proposals()
+    pub fn complete_pending_proposals_job(&self) {
+        let _ = self.log_service.log_info(
+            "Closing proposals".to_string(),
+            Some("close_pending_proposals".to_string()),
```

Comment:
Shouldn't this context (and the other two below) rather be `complete_pending_proposal`?

Reply:
Yep, updated now